app/routes/events/components/EventDetail/index.js

```diff
@@ -138,6 +139,38 @@ export default class EventDetail extends Component<Props> {
       ? () => unfollow(event.isUserFollowing.id, event.id)
       : () => follow(currentUser.id, event.id);
 
+    const infoItems = [
+      event.company && {
+        key: 'Arrangerende bedrift',
+        value: event.company.name
+      },
+      {
+        key: 'Hva',
+        value: EVENT_TYPE_TO_STRING(event.eventType)
+      },
+      {
+        key: 'Når',
+        value: <FromToTime from={event.startTime} to={event.endTime} />
+      },
+      { key: 'Finner sted i', value: event.location },
+      event.activationTime && {
+        key: 'Påmelding åpner',
+        value: <FormatTime time={event.activationTime} />
+      },
+      event.unregistrationDeadline && {
+        key: 'Avregistreringsfrist',
+        value: <FormatTime time={event.unregistrationDeadline} />
+      }
+    ].filter(Boolean);
+
+    const paidItems = [
+      { key: 'Pris', value: `${event.priceMember / 100},-` },
+      event.paymentDueDate && {
+        key: 'Betalingsfrist',
+        value: <FormatTime time={event.paymentDueDate} />
+      }
+    ].filter(Boolean);
```

Comment:
Is there any merit to the idea of moving this filter functionality to the component itself? I imagine doing `someCondition && item` is common enough that most people who use the component will have to add the filter. While it's not a lot of code to write, it might be easy to forget and it might lead to weird error message for those less comfortable with javascript. 

Something like
```
type Props = {
   items: Array<Item | boolean>
 };
...
function InfoList({ items }: Props) {
   return (
     <ul>
       {items.filter(Boolean).map(({ key, value }) => (
 ...
```

Reply:
Thought about it, but it seemed like a bit of a weird API. Can do it though, it's a reasonable argument.